apigee_edge_ui.module

```diff
@@ -168,3 +173,16 @@ function apigee_edge_ui_form_alter(array &$form, FormStateInterface $form_state,
     }
   }
 }
+
+/**
+ * Implements template_preprocess_app_credential_product_list().
+ */
+function apigee_edge_ui_preprocess_app_credential_product_list(array &$variables): void {
+  if (sizeof($variables['content']) <= 5) {
```

Comment:
Can you save the number in a variable at the beginning of the function? So later we can easily alter it to a configuration.